js/version.ts

```diff
@@ -9,7 +9,7 @@ export const version: Version = {
   deno: "",
   v8: "",
   // This string will be replaced by rollup
-  typescript: `ROLLUP_REPLACE_TS_VERSION`
+  typescript: `DENO_REPLACE_TS_VERSION`
```

Comment:
If this value is injected via Rust then `Version` should be removed and `deno --version` should be handled only in Rust (of course in followup PR)

CC @nayeemrmn

Reply:
It’s actually injected in JS...